app/src/main/res/layout/pull_review_message_dialog.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="16dp">
+
+    <com.gh4a.widget.StyleableTextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/pull_request_review_event" />
+
+    <Spinner
+        android:id="@+id/pull_request_review_event"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <com.gh4a.widget.StyleableTextView
+        android:id="@+id/notice"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="8dp"
+        android:text="@string/pull_request_review_description"
+        android:textAppearance="?android:attr/textAppearanceSmall" />
+
+    <EditText
+        android:id="@+id/et_commit_message"
```

Comment:
Needs some text options like auto-capitalized sentences, autocorrection, etc. Look at how it's done in the input field for comments.

Reply:
Done.